bundles/org.openhab.binding.bluetooth.bluez/src/main/java/org/openhab/binding/bluetooth/bluez/internal/DBusBlueZHandlerFactory.java

```diff
@@ -27,22 +27,22 @@
 import org.eclipse.smarthome.core.thing.binding.ThingHandler;
 import org.eclipse.smarthome.core.thing.binding.ThingHandlerFactory;
 import org.openhab.binding.bluetooth.BluetoothAdapter;
-import org.openhab.binding.bluetooth.bluez.BlueZAdapterConstants;
-import org.openhab.binding.bluetooth.bluez.handler.BlueZBridgeHandler;
+import org.openhab.binding.bluetooth.bluez.DBusBlueZAdapterConstants;
+import org.openhab.binding.bluetooth.bluez.handler.DBusBlueZBridgeHandler;
 import org.osgi.framework.ServiceRegistration;
 import org.osgi.service.component.annotations.Component;
 
 /**
- * The {@link BlueZHandlerFactory} is responsible for creating things and thing
+ * The {@link DBusBlueZHandlerFactory} is responsible for creating things and thing
  * handlers.
  *
- * @author Kai Kreuzer - Initial contribution and API
+ * @author Benjamin Lafois - Initial contribution and API
  */
-@Component(service = ThingHandlerFactory.class, configurationPid = "binding.bluetooth.bluez")
-public class BlueZHandlerFactory extends BaseThingHandlerFactory {
+@Component(service = ThingHandlerFactory.class, configurationPid = "binding.bluetooth.dbusbluez")
+public class DBusBlueZHandlerFactory extends BaseThingHandlerFactory {
```

Comment:
The name of this file shouldn't be changed.

```suggestion
@Component(service = ThingHandlerFactory.class, configurationPid = "binding.bluetooth.bluez")
public class BlueZHandlerFactory extends BaseThingHandlerFactory {
```